stdlib/regex.gr

```diff
@@ -1685,18 +2045,23 @@ let rec validate = (re: ParsedRegularExpression, numGroups) => {
   /**
    Computes the range of possible UTF-8 byte lengths for the given character range
    */
-  let rangeUtf8EncodingLengths = (rng) => {
+
+  let rangeUtf8EncodingLengths = rng => {
     let (min, max, _) = List.reduce(((min1, max1, n), (segStart, segEnd)) => {
-      if (rangeOverlaps(rng, segStart, segEnd)) {
-        (min(min1, n), max(max1, n), n + 1)
-      } else {
-        (min1, max1, n + 1)
-      }
-    }, (4, 0, 1), [(0, 127), (128, 0x7ff), (0x800, 0x7fff), (0x10000, 0x10ffff)])
+        if (rangeOverlaps(rng, segStart, segEnd)) {
+          (min(min1, n), max(max1, n), n + 1)
+        } else {
+          (min1, max1, n + 1)
+        }
+      }, (
```

Comment:
Weird that this one broke but not the next one

Reply:
Fixed.